### ISSUE TITLE
Can adding additional search engines allow more accuracy?

### ISSUE THREAD

**Issue by kandyceasania101** [mikealmond/hq-trivia-assistant#79]
I'm not completely familiar with php but in looking at your functions script you search google.ca and ca.yahoo. Do you think adding additional search engines would help getting better answers or would it make the program slower?

**Comment by manofsteel2017** [comment]
yeah acurracy is kinda bad 
wish hquack didn't get shut down. 
if anyone has any secret other bots would be great

**Comment by tombets2424** [comment]
That's a possibility but, yes it would make it slower as it would have to search a wider database of answers. Depending on how it was done, it would also need to display more methods.

**Comment by manofsteel2017** [comment]
yeah method 2 sometimes is more accurate but also takes super longer to get the answers
so right now this is all we got. 
all other sites got shut down
hope they don't come after this one too

**Comment by MelonSmasher** [comment]
I've seen a slight improvement with Wikipedia added to the search engine pool. See #85. I've tried with both Big & Ask and I saw no improvement.

**Comment by MelonSmasher** [comment]
> That's a possibility but, yes it would make it slower as it would have to search a wider database of answers. Depending on how it was done, it would also need to display more methods.

I did not notice a slow down, all of the requests to the search engines are made asynchronously.

**Comment by manofsteel2017** [comment]
can this be added to the hq assistant site since some of us can't run it lol

**Comment by MelonSmasher** [comment]
There's a site? Also I don't control the site if there is one, just a PR into this script.

**Comment by tombets2424** [comment]
I said it could slow down as for some the HQ bot doesn't load method 2 in time. I think it's due to their Internet speed or their pc isn't the best. I guess adding more methods is an option for some people.

**Comment by manofsteel2017** [comment]
hope you can make a more accurate bot, tom

**Comment by anwillcus** [comment]
why cant i add more then 3 search engines?
